source/frontend/src/features/leases/detail/LeasePages/insurance/edit/EditInsuranceContainer.tsx

```diff
@@ -68,48 +70,46 @@ const InsuranceEditContainer: React.FunctionComponent<
       innerRef={formikRef}
     >
       {formikProps => (
-        <>
-          <h2>Required coverage</h2>
-          <div>Select the coverage types that are required for this lease or license.</div>
-
-          <FormSectionClear>
-            <FieldArray
-              name={withNameSpace('visibleTypes')}
-              render={arrayHelpers => (
-                <Form.Group>
-                  {insuranceTypes.map((code: ILookupCode, index: number) => (
-                    <Form.Check
-                      id={`insurance-checbox-${index}`}
-                      type="checkbox"
-                      name="checkedTypes"
-                      key={index + '-' + code.id}
-                    >
-                      <Form.Check.Input
-                        id={'insurance-' + index}
-                        data-testid="insurance-checkbox"
+        <StyledSummarySection>
+          <Section header="Required Coverage">
+            <SectionField label="Select coverage types">
+              <FieldArray
+                name={withNameSpace('visibleTypes')}
+                render={arrayHelpers => (
+                  <Form.Group>
+                    {insuranceTypes.map((code: ILookupCode, index: number) => (
+                      <Form.Check
+                        id={`insurance-checkbox-${index}`}
                         type="checkbox"
                         name="checkedTypes"
-                        value={code.id + ''}
-                        checked={formikProps.values.visibleTypes.includes(code.id + '')}
-                        onChange={(e: any) => {
-                          handleOnChange(e, code, arrayHelpers);
-                        }}
-                      />
-                      <Form.Check.Label htmlFor={'insurance-' + index}>
-                        {code.name}
-                      </Form.Check.Label>
-                    </Form.Check>
-                  ))}
-                </Form.Group>
-              )}
-            />
-          </FormSectionClear>
+                        key={index + '-' + code.id}
+                      >
+                        <Form.Check.Input
+                          id={'insurance-' + index}
+                          data-testid="insurance-checkbox"
+                          type="checkbox"
+                          name="checkedTypes"
+                          value={code.id + ''}
```

Comment:
is this just casting to a string? would .toString() be more explicit?

Reply:
I didn't change this code (only the surrounding section) - but will take a look